code/src/components/StartGame.js

```diff
@@ -0,0 +1,112 @@
+import React, { useState } from 'react'
+import { useDispatch, useSelector, batch } from 'react-redux'
+import styled from 'styled-components'
+
+import { createPlayer } from '../reducers/game'
+import { game } from 'reducers/game'
+import { GameDescription } from './GameDescription'
+import { WelcomePlayer } from './WelcomePlayer'
+import maze from '../assets/maze.svg'
+ 
+const GameContainer = styled.div`
```

Comment:
You could also make another styled component and import the styles here. Just a recommendation.